src/components/TodoItems.jsx

```diff
@@ -0,0 +1,81 @@
+import { categoryList, priorityList } from '../constants'
+import './todo-items.css'
+
+export default function TodoItems(props) {
```

Comment:
We can destructure all required props here

```js
export default function TodoItems({
  markAsDone,
  handleOnChange
}) {
```

so that we don't have to write `props.` everywhere.